source/_static/js/redirects.js

```diff
@@ -375,8 +375,10 @@ removedUrls['4.9'] = [
   '/user-manual/capabilities/policy-monitoring/rootcheck/index.html',
   '/user-manual/capabilities/policy-monitoring/rootcheck/rootcheck-configuration.html',
   '/user-manual/capabilities/policy-monitoring/rootcheck/rootcheck-faq.html',
-  '/user-manual/capabilities/policy-monitoring/ciscat/ciscat.html',
   '/user-manual/capabilities/malware-detection/osquery.html',
+  '/user-manual/capabilities/policy-monitoring/ciscat/ciscat.html',
+  '/user-manual/reference/ossec-conf/wodle-ciscat.html',
+  '/user-manual/reference/ossec-conf/wodle-openscap.html',
```

Comment:
Shouldn't this be removed?

Reply:
These lines specifically remove those pages. These lines are entries of the `removedUrls['4.9']` array.